tools/CACertificates/certGen.sh

```diff
@@ -367,7 +367,7 @@ function generate_verification_certificate()
 ###############################################################################
 # Generates a certificate for a device, chained to the intermediate.
 ###############################################################################
-function generate_device_certificate()
+function generate_identity_device_certificate()
```

Comment:
Should this function by `generate_device_identity_certificate`?

Reply:
Agreed and fixed.